examples/templates/vue/src/main.js

```diff
@@ -0,0 +1,8 @@
+import Vue from 'vue'
+import App from './App.vue'
+
+Vue.config.productionTip = false
```

Comment:
An ENV should be used? Are you sure that the Vue examples are built in the production mode?

Reply:
Oh, right. Nervermind then.